frontend/src/js/controllers/challengeHostTeamsCtrl.js

```diff
@@ -369,6 +369,82 @@
             }, function() {});
         };
 
+// Delete a particular host
+vm.confirmMemberDelete = function(ev, hostTeamId, hostId) {
+    ev.stopPropagation();
+    // Appending dialog to document.body to cover sidenav in docs app
+    var confirm = $mdDialog.confirm()
+        .title('Would you like to remove this member?')
+        .textContent('Note: This action will remove this member from the team.')
```

Comment:
change this `Note: This action will remove this member from the team.` to `Note: This action will remove the member from team.`